### PR TITLE
Add OpenAPI yaml spec for Load endpoint

### PR DESCRIPTION
**Pull request by annaleeli**
Add OpenAPI Load endpoint for only cluster load as partition load is accessed via a different path and capacity is not yet merged.

### CODE REVIEW

File: cruise-control/src/yaml/endpoints/load.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  get:
+    operationId: load
+    summary: Get the cluster load once Cruise Control Load Monitor is running.
+    parameters:
+      - name: start
+        in: query
+        description: Start time of the cluster load.
+        schema:
+          type: long
+          format: int64
```
Comment (kidkun) [inline cruise-control/src/yaml/endpoints/load.yaml]: can we add a `minimum` and `default` field for this parameter? (see [partitionLoad.yaml](https://github.com/linkedin/cruise-control/blob/b15fbffd04b8bd4fa561e13a3bc247f02fc794dc/cruise-control/src/yaml/endpoints/partitionLoad.yaml))

File: cruise-control/src/yaml/endpoints/load.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          # Default is time of earliest valid window.
+      - name: end
+        in: query
+        description: End time of the cluster load.
+        schema:
+          type: long
+          format: int64
```
Comment (kidkun) [inline cruise-control/src/yaml/endpoints/load.yaml]: can we add a `minimum` field for this parameter? (see [partitionLoad.yaml](https://github.com/linkedin/cruise-control/blob/b15fbffd04b8bd4fa561e13a3bc247f02fc794dc/cruise-control/src/yaml/endpoints/partitionLoad.yaml))

File: cruise-control/src/yaml/endpoints/load.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          # Default is current system time.
+      - name: time
+        in: query
+        description: End time of the clutser load.
+        schema:
+          type: long
+          format: int64
```
Comment (kidkun) [inline cruise-control/src/yaml/endpoints/load.yaml]: can we add a `minimum` and `default` field for this parameter?

Comment (annaleeli) [reply cruise-control/src/yaml/endpoints/load.yaml]: I added `minimum` for this parameter as per the comment. However, `default` for this field is the current time stamp at time of execution (which is difficult to express as a constant value). I omitted adding the field in the yaml.

Let me know if you have a strong preference for `default` here and I will investigate a way to add it in.

**Comment by annaleeli** [comment]
@kidkun Thank you for the review! I addressed the comments you have left on the PR.

**Pull request merged**